install.sh

```diff
@@ -4,9 +4,7 @@
 mkdir -p $HOME/.tdk/bin
 
 # Fetch the latest JAR file
-# FIXME: select latest version automatically 
-# LATEST_JAR_URL=$(curl -s https://synthesizedio.jfrog.io/artifactory/tdk-public/ | grep -o 'href="[^"]*.jar"' | sed 's/href="//;s/"//' | grep -E '_v[0-9]+\.[0-9]+\.[0-9]+\.jar$' | sort -t. -k2,2n -k3,3n -k4,4n | tail -n 1)
-LATEST_JAR_URL="synthesized_tdk_v1.100.0.jar"
+LATEST_JAR_URL="synthesized_tdk_latest.jar"
```

Comment:
nit: I would introduce TDK_VERSION variable with a default value so you can override if needed. also, the variable name is misleading, it's not an URL in fact:

```bash
TDK_VERSION=${TDK_VERSION:-"latest"}
TDK_JAR="synthesized_tdk_${TDK_VERSION}"

curl -L -o $HOME/.tdk/latest.jar "https://governor-static.synthesized.io/jars-public/$TDK_JAR"
```